ksql-engine/src/main/java/io/confluent/ksql/function/udf/PluggableUdf.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.ksql.function.udf;
+
+import io.confluent.ksql.function.UdfInvoker;
+
+/**
+ * Class to allow conversion from Kudf to UdfInvoker.
```

Comment:
Why would we want to retairn `Kudf` in the longer term? Using the strongy typed annotated classes seems much better IMO.

Reply:
Yes - in the long term we will hopefully remove it, but this is an interim step so that we can maintain the current `Kudf`s and the new. Once we move all the existing `Kudf`s we'll be able to remove this.